src/components/BackButton/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { useHistory } from 'react-router';
+
+const BackButton = () => {
+	const history = useHistory();
+	return (
+		<>
+			<input id="back-btn" onClick={history.goBack} type="button" value="<"></input>
```

Comment:
why did you use an input and not a button, does it make things easier? I mean they're both the same thing anyway im literally just asking out of curiosity

Reply:
bc the value of the button was "<" it threw an error when I used it in the button